Add installation steps for Flutter archive downloads (#13325)
flutter/website: src/content/install/archive.md

[calendar versioning]: https://calver.org/
[Flutter SDK versioning]: {{site.repo.flutter}}/blob/main/docs/releases/Release-versioning.md

## Install from an archive

> After downloading a Flutter SDK archive from this page, follow these steps to install it:

Reviewer: ```suggestion
After downloading a Flutter SDK archive from this page,
use the following steps to install:
```

1. Extract the downloaded archive to a desired location on your system.
2. Add the Flutter `bin` directory to your system's `PATH`.
3. Run the following command to verify the installation:

```bash id="yfl9s6"
flutter doctor
```

> The `flutter doctor` command checks your environment and displays a report of the status of your Flutter installation.

Reviewer: ```suggestion
The `flutter doctor` command checks your environment
and reports on the status of your installation.
```

> For more detailed platform-specific setup instructions, refer to the main [Flutter installation guide](../install).

Reviewer: ```suggestion
For detailed platform-specific setup instructions,
refer to the main [Flutter installation guide](../install).
```
